libs/sdk-ui-tests-e2e/scripts/run_boiler_app.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# (C) 2021-2023 GoodData Corporation
+
+source /workspace/common/scripts/ci/utils.sh
+
+trap "npm run delete-ref-workspace || true" EXIT
+
+npm run create-ref-workspace
+export TEST_WORKSPACE_ID=$(grep TEST_WORKSPACE_ID .env | cut -d '=' -f 2)
+
+apt-get update && apt-get install -y curl && apt-get install -y jq && apt-get install -y sed && apt-get clean
+
+npx --yes @gooddata/$BOILER_APP_VERSION init $BOILER_APP_NAME --language $SDK_LANG
+pushd $BOILER_APP_NAME
+
+tmp=$(mktemp)
+jq --arg host "${HOST}" --arg ws "${TEST_WORKSPACE_ID}" --arg backend "${sdk_backend}" \
+'.gooddata.hostname = $host | .gooddata.workspaceId = $ws | .gooddata.backend = $backend' \
+package.json > $tmp
+mv $tmp package.json
+
+# work around for RAIL-4988, modify backend.ts for Bear backend
+if [[ "$sdk_backend" == 'bear' ]]; then
+export GDC_USERNAME=${USER_NAME:?}
+export GDC_PASSWORD=${PASSWORD:?}
+
+cat > src/backend.${SDK_LANG} <<-EOF
```

Comment:
If this status file, create it as a file, dont use `cat`. Only use `cat` if we're going to put some variables in its content

Reply:
changed to copy file